.github/workflows/wheel.yml

```diff
@@ -0,0 +1,100 @@
+name: wheels
+on:
+  pull_request:
+  push:
+    branches: [master, test-me-*]
+    tags: ['*']
```

Comment:
we _may_ want to only run this on tags, it takes a lot of builder slots and the windows cpython2.7 takes forever -- to do that one would remove lines 3 and 5

Reply:
Yeah I will do that once it's all working. Manual trigger would be fine too. I think we'd have to add `workflow_dispatch:` for that?

Reply:
since this build worked, I'm going to amend this to just `tags` and `workflow_dispatch`

Reply:
oh! I was too slow, hehe nbd I'll let you handle it then!